crates/ide_assists/src/handlers/extract_function.rs

```diff
@@ -109,10 +109,15 @@ pub(crate) fn extract_function(acc: &mut Assists, ctx: &AssistContext) -> Option
 
             let new_indent = IndentLevel::from_node(&insert_after);
             let old_indent = fun.body.indent_level();
+            let is_body_contains_await = body_contains_await(&fun.body);
```

Comment:
Let's call this body_has_await or something like that. I don't know if it can have the same name as the function.